app/screens/settings/notifications/notifications.tsx

```diff
@@ -62,6 +65,21 @@ const Notifications = ({
     const serverUrl = useServerUrl();
     const notifyProps = useMemo(() => getNotificationProps(currentUser), [currentUser?.notifyProps]);
     const callsRingingEnabled = useMemo(() => getCallsConfig(serverUrl).EnableRinging, [serverUrl]);
+    const [isNotRegistered, setIsNotRegistered] = useState(false);
```

Comment:
Negative names are harder to reason with. I suggest changing the name (and values around) to `isRegistered`.